testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -3651,6 +3651,43 @@ QUnit.module('search substitution', {
         assert.equal(loadMock.callCount, 0, 'items are not loaded');
     });
 
+    QUnit.test('items should be loaded after the selection was changed and removed on the \'backspace\' key press (T1058326)', function(assert) {
```

Comment:
let's use double instead of `/'`. It simplifies search when you copy test name from test page